src/state/reducer.js

```diff
@@ -79,55 +77,38 @@ const move = ({
 
   const previous: CurrentDrag = state.drag.current;
   const initial: InitialDrag = state.drag.initial;
-  const droppable: DroppableDimension = state.dimension.droppable[initial.source.droppableId];
+  const currentWindowScroll: Position = windowScroll || previous.windowScroll;
```

Comment:
rather than everywhere calculating page - it does just done here based on the window scroll